Apps/PackageTest/0.64.0/package-lock.json

```diff
@@ -8,7 +8,7 @@
       "name": "PackageTest",
       "version": "0.0.1",
       "dependencies": {
-        "@babylonjs/core": "5.2.0",
+        "@babylonjs/core": "5.6.1",
```

Comment:
It seems weird that only one line in this file changed, and there isn't a hash update somewhere. Probably won't result in any issues, but kinda unexpected.